WPFHospitalEditor/Building.xaml.cs

```diff
@@ -16,8 +16,8 @@ namespace WPFHospitalEditor
     /// </summary>
     public partial class Building : Window
     {
-        private Dictionary<int, Floor> buildingFloors = new Dictionary<int, Floor>();
-        public List<MapObject> floorBuildingObjects;
+        public Dictionary<int, Floor> buildingFloors = new Dictionary<int, Floor>();
+        //public List<MapObject> floorBuildingObjects;
```

Comment:
You left comments again, here and on 33rd line.

Reply:
Will be fixed after Boža's review